### ISSUE TITLE
Code Table Request - new specimen part name: braincase

### ISSUE THREAD

**Issue by KatherineLAnderson** [ArctosDB/arctos#7754]
### Goal
Add "braincase" with search term "neurocranium" to part names code table

### Context
There is not an existing term that captures this part. Braincase is referenced in other definitions of various skull bones, but itself is not in the code table. 
Other context: "Cranium" is **not** synonymous with braincase. Definition needs to be cleaned up--I can create another issue for that.

### Table
https://arctos.database.museum/info/ctDocumentation.cfm?table=CTSPECIMEN_PART_NAME

### Proposed Value
braincase with search term neurocranium

### Proposed Definition
Also called the neurocranium, the part of the skull/cranium that encases the brain. The braincase is comprised of different bones/areas of the skull in different vertebrates. 
https://en.wikipedia.org/wiki/Neurocranium

### Collection type
paleo, bio

#### Attribute data type
n/a

#### Attribute controlled values
n/a

#### Attribute units
n/a

#### Part preservation attribute affect on "tissueness"
n/a

### Priority
priority high--needed for data migration

### Available for Public View
yep

### Helpful Actions

- [x ] Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184).

- [ ] Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

@ArctosDB/arctos-code-table-administrators 

## Approval
_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection
_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation
_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

- [ ] Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel.

- [ ] Add or revise the code table term/definition as described above. Ensure the URL of this Issue is included in the definition. URLs should be included *_as text_*, separated by spaced pipes. Do not include HTML in definitions.

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by dustymc** [comment]
I'm not sure this is compatible with https://docs.google.com/document/d/1mTWY6HRlC3TSrAb38W2HOQHiVO2F1tA1m5jaLstYMS8/edit#heading=h.w0tvdqpqd23q

**Comment by KatherineLAnderson** [comment]
> > > "Cranium" is not synonymous with braincase. Definition needs to be cleaned up--I can create another issue for that.
> > 
> > 
> > Would using cranium with condition partial and braincase in part remark work?
> 
> Probably... but this is another one for the parts-part discussion, I think. My main concern about remarks is that people (curators, students, whoever is doing data entry) won't know where to put this and it'll end up in 10 different places. If it's controlled vocab (i.e., parts or parts-part) there's direction in the documentation and code tables.

I just started to do it this way because I need to get the data in. I think there's still need for discussion as part of parts-parts #7763 , and also issue #7755 is relevant.

**Comment by dustymc** [comment]
I think this is compatible with https://docs.google.com/spreadsheets/d/1-QC9L8jbOzaG7vstp5iHyx8ivlyvr3BrtiCY_GBimAg/edit?gid=0#gid=0&range=92:92, but

1. The wiki doc link suggests the part and search term should be the other way around; is there some guidance (which could be added to the table documentation) or rhyme or reason for choosing a synonym when such things are available?
2. ontology link please?

**Comment by KatherineLAnderson** [comment]
>     1. The wiki doc link suggests the part and search term should be the other way around; is there some guidance (which could be added to the table documentation) or rhyme or reason for choosing a synonym when such things are available?
> 

Paleontologists use braincase. I'm assuming other kinds of -ologists use neurocranium. If it's one or the other, as long as the other is included as a synonym and search term, that would be OK by me.

>     2. ontology link please?

Does this work? Again, it's filed as neurocranium with braincase as a synonym, so perhaps the paleontologists are overruled.

http://purl.obolibrary.org/obo/UBERON_0001703

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Googling around and flipping through old textbooks, I'm seeing it used both ways. I'd say let's just defer to how uberon has it, neurocranium with synonym braincase.
